### DISCUSSION TITLE
Baby is born !!

### DISCUSSION THREAD

**Discussion by Romanitho** [Romanitho/Winget-AutoUpdate#284] [General]
Hi all,

Sorry to be less present on this project right now. Just a quick update!! I'm a dad now 🍾😇  
My little girl is born on 8th of Feb and everybody is fine.

As you can guess, my free time now is close to 0 😅 but I'll try to follow this as much as I can.

With Love,

Romain.

**Comment by KnifMelti** [comment]
A real @Romanitho then... ❤️👍
